src/components/OverflowTooltip/OverflowTooltip.tsx

```diff
@@ -59,7 +65,7 @@ const OverflowTooltip: React.FC<OverflowTooltipProps> = ({
       <Tooltip.Content side="top">
         <Tooltip.Arrow />
         <Tooltip.ContentHeading>{header}</Tooltip.ContentHeading>
-        {title ?? children}
+        <div className={classes.children}>{title ?? children}</div>
```

Comment:
suggestion: Text component from new macaw support ellipsis

Reply:
I don't want to risk adding it here and breaking possibly all other places where the Tooltip is used now